Add generic text and grammar revisions

srschramm opened the pull request:
Moving my minor textual changes from sharelatex to github.

srschramm committed:
Textual fixes from Steven on July 8

matthewfeickert commented:
Thanks, @srschramm (that was quick :+1:). This builds the draft that can [be found here](https://feickert.web.cern.ch/feickert/IML-CWP/PRs/draft_ml-cwp_PR24.pdf). After a quick scan this can get merged in right away to serve as the basis for the PR that @pseyfert and @SergeiML can write together for your Issue #23.

srschramm commented:
Hi Matthew,

I just looked at the build draft and it looks fine and as expected from my side.  I agree that this can be merged right away and serve as the basis for issue #23.  That was indeed the intent - fix the trivial things, and let the more complex ones be debated.

Best regards,
Steven

matthewfeickert commented:
@srschramm Actually, let me go in and fix the issues with the `\clearpage` now. I know you need to go to bed, but I'll get this and then let @pseyfert and @SergeiML do the other revisions.

srschramm commented:
Great, thanks Matthew.  Indeed, I'll head to bed now.  Good luck to @pseyfert and @SergeiML for the other fixes!  If you're happy with my proposed solutions it shouldn't take too much work (I hope)!

matthewfeickert commented on src/collaboration.tex:
```diff
@@ -30,7 +30,7 @@ \subsection{Academic Outreach and Engagement}
 \clearpage
```
This `\clearpage` was put in originally for page continuity, but now needs to get edited.

matthewfeickert committed:
Make figures and tables fit with text

Avoid cutting off single trailing lines of paragraphs from appearing
orphaned due to a table or figure appearing at the start or end of a
page.
This effectivley removes most uses of \clearpage in the document.
